streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedCachingPersistentWindowStoreTest.java

```diff
@@ -90,8 +88,15 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertThrows;
 import static org.junit.Assert.assertTrue;
-
-@RunWith(Parameterized.class)
```

Comment:
We are removing the ability to run the tests in a parameterized manner. Hence, this test will not for parameters defined in 
```
@Parameterized.Parameters(name = "{0}")
    public static Collection<Object[]> data() {
        return asList(new Object[][] {
            {true},
            {false}
        });
    }
```

I understand that you did this because we cannot have two `RunWith` annotations. You can alternatively use 
```
@Rule
    public MockitoRule mockito = MockitoJUnit.rule().strictness(Strictness.STRICT_STUBS);
```
instead of the `@RunWith(MockitoJUnitRunner.StrictStubs.class)`

Reply:
Looks like Rule is also not needed.